canonical-web-and-design/ubuntu.com
# Rename "deploy" to "konf"

## Done

- Renamed the /deploy/ directory to /konf/

## QA

- Check out this feature branch
- Ensure that the name change is reflected


## Issue / Card

Fixes web-squad#2984
